pkg/operator/k8sutil/node.go

```diff
@@ -168,9 +168,27 @@ func NodeMeetsPlacementTerms(node v1.Node, placement rookv1.Placement, ignoreWel
 	if !NodeIsTolerable(node, placement.Tolerations, ignoreWellKnownTaints) {
 		return false, nil
 	}
+	if !NodeHasTopologyKeys(node, placement.TopologySpreadConstraints) {
+		return false, nil
+	}
 	return true, nil
 }
 
+// NodeHasTopologyKeys returns true if the node has the specified topology label.
+// If `whenUnsatisfiable` is `ScheduleAnyWay`, the constraint is ignored.
+func NodeHasTopologyKeys(node v1.Node, constraints []v1.TopologySpreadConstraint) bool {
```

Comment:
No need for this method either according to the other comment.

Reply:
Removed, thanks.